contracts/crypto-verify/README.md

```diff
@@ -10,15 +11,17 @@ Input formats are serialized byte slices for Message, Signature, and Public Key.
 
 ### secp256k1:
 
-- Message: A serialized message. It will be hashed by the contract using SHA-256, and the hashed value will be fed to the
-  verification function.
-- Signature:  Serialized signature, in "compact" Cosmos format (64 bytes). Ethereum DER needs to be converted.
-- Public Key:  Compressed (33 bytes) or uncompressed (65 bytes) serialized public key, in SEC format.
+- Message: A serialized message. It will be hashed by the contract using
+  SHA-256, and the hashed value will be fed to the verification function.
+- Signature: Serialized signature, in "compact" Cosmos format (64 bytes).
+  Ethereum DER needs to be converted.
+- Public Key: Compressed (33 bytes) or uncompressed (65 bytes) serialized public
+  key, in SEC format.
```

Comment:
What tool / method do you use to format the docs, by the way?

I've tried adding EditorConfig support to IntelliJ, but it's confusing, and I may have it badly configured.

Reply:
EditorConfig is only the very baseline confit for formatting. On top of that I use [prettier](https://prettier.io/). You can run it via `./devtools/format_md.sh`